setup/module/api/policies/elb_group_policy.json

```diff
@@ -1,21 +1,10 @@
 {
     "Version": "2012-10-17",
     "Statement": [
-	{
-            "Effect": "Allow",
-            "Action": [
-                "elasticloadbalancing:Describe*"
-            ],
-            "Resource": "*"
-        },
         {
             "Effect": "Allow",
-            "Action": [
-                "elasticloadbalancing:*"
-            ],
-            "Resource": [
-		"arn:aws:elasticloadbalancing:${region}:${account_id}:loadbalancer/l0-${name}-*"
-	    ]
+            "Action": "elasticloadbalancing:*",
+            "Resource": "*"
```

Comment:
I'm a little concerned about this change because it opens up the door for the IAM user to manipulate other load balancers in the AWS account when it shouldn't have permission to do so.

Reply:
I do share your concern but the IAM User needs almost all elasticlaodbalancing permissions anyhow where resource=`*`. I'm not really sure what we're trying to prevent here.

Reply:
Was there something specific that was breaking with the way it was before? It looks like it had star privileges on a narrow band of load balancer resources, but is there something fundamentally different about the ARN with respect to application load balancers?

Reply:
There was nothing breaking with the way it was before. But in response to a review comment, https://github.com/quintilesims/layer0/pull/580/files/b651a4094bf146d4957fa3ff2b84b11cde2f941c#diff-e52a5f1d1c193669b1a7570d8eab8485

I gave all elasticloadbalancing permissions. So it didn't make sense to have 2 statements with the same permissions, one with resource="*" and the other with resource="arn:aws:elasticloadbalancing:${region}:${account_id}:loadbalancer/l0-${name}-*"

Reply:
I believe the permissions for `resource="*"` was `Describe*` and the permission for the longer ARN was star permissions.

Reply:
It was `Describe*` prior to this pr. After this PR, it was initially a list of 20+ permissions which is now `elasticloadbalancing:*`.

Reply:
I see. I think overall my comment might be for a discussion for another issue. I think the permissions can be designed in a way that can balance readability and security, but for now I'll just :heavy_check_mark: this.